### ISSUE TITLE
add class_name option to Contract.create

### ISSUE THREAD

**Issue by gen1321** [EthWorks/ethereum.rb#74]
Hey! first of all thx for great work on this gem.

 So i think `Ethereum::Contract.create` should accept `class_name` additionally to just `name` because if i have contract in my truffle project with same name as my model it will simple replace it.

 For example i have `AmazingCoin` it my truffle project and `AmazingCoin` in my rails app when i will call `Ethereum::Contract.create(name: "AmazingCoin", truffle: { paths: [ './coin' ]} ...)` it will replace my Rails model and make it inaccessible. 

So it think good solution here is to add additional parameter to `create`, such as `class_name`. If `class_name`  passed, we obviously create class with `class_name` instead of just `name`, otherwise use `name` as class name . Will u accept pr if i make one?

**Comment by jasonadkison** [comment]
I noticed this too, and it's not good. The gem creates the contract class as a global constant. The simplest fix would be to create it on the Ethereum::Contract class. I have [this](https://github.com/jasonadkison/ethereum.rb/commit/8adc037edc59933e011fdb7bf56fa12f0f855323) potential solution I've been using in production. Any thoughts?

**Comment by gen1321** [comment]
Yeah this make sense! i think you should make PR :) but i wonder it there any case when we want to customize class name?

**Comment by jasonadkison** [comment]
Adding the option to name the class through an argument sounds like a good feature to have to me. Nonetheless I've opened a PR for what we discussed here: https://github.com/EthWorks/ethereum.rb/pull/75
